### PR TITLE
SUS-2200 in rename user tool use rollup_wiki_user_event instead rollup_edit_events

### PR DESCRIPTION
**Pull request by sqreek**
https://wikia-inc.atlassian.net/browse/SUS-2200

@mixth-sense @macbre

### CODE REVIEW

File: extensions/wikia/UserRenameTool/RenameUserHelper.class.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ static public function lookupRegisteredUserActivity( $userID ) {
 		if ( empty( $wgDevelEnvironment ) ) { // on production
```
Comment (WikiaTech) [inline extensions/wikia/UserRenameTool/RenameUserHelper.class.php:34]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [RenameUserHelper.class.php#L51](https://github.com/Wikia/app/blob/18f29e003dd1185e0f91717f0e4079605a387c16/extensions/wikia/UserRenameTool/RenameUserHelper.class.php#L51): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)
1. ![INFO][INFO] [RenameUserHelper.class.php#L101](https://github.com/Wikia/app/blob/18f29e003dd1185e0f91717f0e4079605a387c16/extensions/wikia/UserRenameTool/RenameUserHelper.class.php#L101): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Review by macbre — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Per my comment on the JIRA case, let's use `stats.events` so that there is no delay.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [RenameUserHelper.class.php#L51](https://github.com/Wikia/app/blob/6a710df757ab2c1df88d4030fd9efa4f70cd93f9/extensions/wikia/UserRenameTool/RenameUserHelper.class.php#L51): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)
1. ![INFO][INFO] [RenameUserHelper.class.php#L101](https://github.com/Wikia/app/blob/6a710df757ab2c1df88d4030fd9efa4f70cd93f9/extensions/wikia/UserRenameTool/RenameUserHelper.class.php#L101): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Review by ghost — APPROVED** [review]

**Pull request merged**